packages/twenty-server/src/modules/workflow/workflow-runner/workflow-runner.job.ts

```diff
@@ -1,32 +1,54 @@
+import { Process } from 'src/engine/integrations/message-queue/decorators/process.decorator';
 import { Processor } from 'src/engine/integrations/message-queue/decorators/processor.decorator';
 import { MessageQueue } from 'src/engine/integrations/message-queue/message-queue.constants';
-import { Process } from 'src/engine/integrations/message-queue/decorators/process.decorator';
-import { WorkflowRunnerService } from 'src/modules/workflow/workflow-runner/workflow-runner.service';
+import { TwentyORMGlobalManager } from 'src/engine/twenty-orm/twenty-orm-global.manager';
+import { WorkflowWorkspaceEntity } from 'src/modules/workflow/common/standard-objects/workflow.workspace-entity';
 import { WorkflowCommonService } from 'src/modules/workflow/common/workflow-common.services';
+import { WorkflowRunnerService } from 'src/modules/workflow/workflow-runner/workflow-runner.service';
 
-type RunWorkflowJobData = { workspaceId: string; workflowVersionId: string };
+export type RunWorkflowJobData = {
+  workspaceId: string;
+  workflowId: string;
+  payload: object;
+};
 
 @Processor(MessageQueue.workflowQueue)
 export class WorkflowRunnerJob {
   constructor(
     private readonly workflowCommonService: WorkflowCommonService,
     private readonly workflowRunnerService: WorkflowRunnerService,
+    private readonly twentyORMGlobalManager: TwentyORMGlobalManager,
   ) {}
 
   @Process(WorkflowRunnerJob.name)
   async handle({
     workspaceId,
-    workflowVersionId,
+    workflowId,
+    payload,
```

Comment:
I think we should be consistent between those `handle` function parameters and the `runWorkflow`parameters. Lets use the workflowId in `runWorkflow` then. Also, we should update the `workflowCommonService.getWorkflowVersion` so that we don't need `twentyORMGlobalManager` service here
Then the `runWorkflow` method can also use the updated version of `workflowCommonService.getWorkflowVersion`

Reply:
@martmull agreed with the input update. But thinking again about the endpoint, I think this is the version we want to run. This endpoint will be used to test versions. I rename it to be consistent. Happy to discuss it if you disagree!